fmriprep/cli/parser.py

```diff
@@ -40,10 +40,17 @@ def _drop_sub(value):
         value = str(value)
         return value.lstrip('sub-')
 
+    def _filter_pybids_none_any(dct):
+        import bids
+        for k, v in dct.items():
+            if v == '__any__':
+                dct[k] = bids.layout.Query.ANY
+        return dct
```

Comment:
The switch to `"*"` seems to have gotten lost in the merge.

```suggestion
        return {k: bids.layout.Query.ANY if v == "*" else v
                for k, v in dct.items()}
```